app/views/accountants/_table_content.html.erb

```diff
@@ -1,22 +1,22 @@
 <table class="table border-side">
-  <table class="table border-side">
-    <thead>
-      <tr>
-        <th>Clinic Name</th>
-        <th>PT initials</th>
-        <th><%= FUND %> payout</th>
-        <th>Weeks at Procedure</th>
-        <th>Pledge Sent Date</th>
-        <th>Check #</th>
-        <th>Date of Check</th>
-        <th></th>
+  <thead>
+    <tr>
+      <th><%= t 'accountants.table_content.clinic' %></th>
+      <th><%= t 'accountants.table_content.pt_initials' %></th>
+      <th><%= t 'accountants.table_content.fund_payout', fund: FUND %></th>
+      <th><%= t 'accountants.table_content.weeks_at_procedure' %></th>
+      <th><%= t 'accountants.table_content.pledge_sent' %></th>
+      <th><%= t 'accountants.table_content.check_number' %></th>
+      <th><%= t 'accountants.table_content.date_of_check' %></th>
```

Comment:
Suggest moving most of these to a shared - I just ran into most of them in the patient pledge PR! #1621 

If they don't feel like a good case for a 'shared' namespace, I think it's more appropriate to name them for the pledge tree, but I don't feel strongly.

Reply:
These might be specialized to accountants, but if they're not I'll move them to the mongoid attributes namespace. Great call.

Reply:
After taking a stab at this the language is like EVER so slightly different, in such a way that I don't think this is a free basket rn. My vote is that we go with the redundancy for now in the name of expedience.

Reply:
(I will happily commit to cleaning this mess up.)